draft-ietf-quic-transport.md

```diff
@@ -1036,9 +1036,9 @@ be used again and requests that the peer replace it with a new connection ID
 using a NEW_CONNECTION_ID frame.
 
 As discussed in {{migration-linkability}}, each connection ID MUST be used on
-packets sent from only one local address.  An endpoint that migrates away from a
-local address SHOULD retire all connection IDs used on that address once it no
-longer plans to use that address.
+packets sent from only one local address, and MUST NOT be used across multiple
+paths that are opened intentionally.  An endpoint SHOULD retire connection IDs
+as they become unusable.
```

Comment:
This new text is less accurate.  It is potentially always possible to reuse connection IDs, but what matters is whether the endpoint forsees use of the connection ID.  The original text is probably not as good as it could be, but that seems unrelated to the stated goal of the PR.

As this is a restatement of the original text in another section, I would prefer to keep this terse.  In fact, I would not use normative language and instead say:

> As discussed in {{blah}}, endpoints limit the use of a connection ID to a single network path where possible.  Endpoints SHOULD retire connection IDs when no longer actively using the network path on which the connection ID was used.

Reply:
I'd tend to agree here.

I think the normative text in question is: 
> ##### Section 9.5 Privacy Implications of Connection Migration
> ...
> An endpoint MUST use a new connection ID if it initiates connection migration as
described in {{initiating-migration}} or probes a new network path as described
in {{probing}}.  An endpoint MUST use a new connection ID in response to a
change in the address of a peer if the packet with the new peer address uses an
active connection ID that has not been previously used by the peer.

This seems to me like: 
> each connection ID MUST be used on packets sent from only one local address

still has value as normative text (although it would likely be better all in one place).

Not sure the new text improves on what's there now, does:
> , and MUST NOT be used across multiple paths that are opened intentionally.

add anything to clarity of this point?

Reply:
Ah, I see later some of this is covered :)